zhaquirks/tuya/ts130f.py

```diff
@@ -0,0 +1,104 @@
+"""Device handler for loratap TS130F smart curtain switch."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import Basic, Groups, OnOff, Ota, Scenes, Time
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+ATTR_CURRENT_POSITION_LIFT_PERCENTAGE = 0x0008
+CMD_GO_TO_LIFT_PERCENTAGE = 0x0005
+
+
+class TuyaWithBacklightOnOffCluster(CustomCluster):
+    """TuyaSmartCurtainOnOffCluster: fire events corresponding to press type."""
+
+    cluster_id = OnOff.cluster_id
+
+    LIGHT_MODE_1 = {0x8001: 0}
+    LIGHT_MODE_2 = {0x8001: 1}
+    LIGHT_MODE_3 = {0x8001: 2}
+
+    attributes = {0x8001: ("backlight_mode", t.enum8)}
+
+
+class TuyaCoveringCluster(CustomCluster, WindowCovering):
+    """TuyaSmartCurtainWindowCoveringCluster: Allow to setup Window covering tuya devices."""
+
+    attributes = WindowCovering.attributes.copy()
+    attributes.update({0xF000: ("tuya_moving_state", t.enum8)})
+    attributes.update({0xF001: ("calibration", t.enum8)})
+    attributes.update({0xF002: ("motor_reversal", t.enum8)})
+
+    def _update_attribute(self, attrid, value):
+        if attrid == ATTR_CURRENT_POSITION_LIFT_PERCENTAGE:
+            # Invert the percentage value (cf https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3757)
+            value = 100 - value
+        super()._update_attribute(attrid, value)
+
+    async def command(
+        self, command_id, *args, manufacturer=None, expect_reply=True, tsn=None
+    ):
+        """Override default command to invert percent lift value."""
+        if command_id == CMD_GO_TO_LIFT_PERCENTAGE:
```

Comment:
Could you try this instead
```suggestion
    async def go_to_tilt_percentage(
        self, *args, manufacturer=None, expect_reply=True, tsn=None
    ):
        """Override default command to invert percent lift value."""
```

It is going to override just that command.